hw2/css/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+	background-color: #f9f9f9;
+	font-size: 13px;
+	font-family: OpenSans, Ubuntu, Antic, sans-serif;
+} 
+.wrapper {
+	margin: auto;
+	padding: 50px 48px 60px 46px;
+	width: 1060px;
+	background-color: #ffffff;
+	box-shadow: 0 5px 5px rgba(0, 0, 0, 0.3);
+}
+/*HEADER*/
+.top-menu {
+	display: flex;
+	background-color: #5e5e5e;
+	background-image: linear-gradient(to top, #444343 0%, #5e5e5e 100%); 
+	margin-top: 40px;
+}
+.top-menu_item {
+	display: block;
+	padding: 14px 25px 11px;
+	border-left: 1px solid rgba(255,255,255,0);
+	border-right: 1px solid rgba(255,255,255,0);
+	text-shadow: 1px 1px 1px rgba(0,0,0,0.5);
+	color: #ffffff;
+	font-size: 15px;
+}
+.top-menu_item:hover {
+	background-image: linear-gradient(to top, #2e2e2e, #4a4a4a)	;
+    border-left: 1px solid rgba(255,255,255,1);
+    border-right: 1px solid rgba(255,255,255,1);
+}
+/*CONTAINER*/
+.container {
+	padding-top: 40px;
+	padding-bottom: 60px;
+}
+.container::after {
+	content: '';
+	display: block;
+	clear: both;
+}
+.left-menu {
+	float: left;
+	width: 240px;
+}
+.left-menu_item {
+	border: 1px solid #c8c8c8;
+	border-top: none;
+}
+.left-menu_item:first-child {
+	border-top: 1px solid #c8c8c8;
+}
+.left-menu_link {
+	display:block;
+	color: #656363;
+	padding: 15px 0px 11px 22px;
+}
+.left-menu_item:hover {
+	background-color: #f9f9f9;
+}
+.left-menu_link:hover {
+	color: #2c2c2c;
+	position: relative;
+	left: 20px;
+}
+.content {
+	float: right;
+	width: 678px;
+}
+.image {
+	padding: 47px 89px 46px;
+	border: 1px solid #c8c8c8;
+}
+.article {
+	margin: 39px 29px 0px 24px;
+	color: #2c2c2c;
+}
+/*FOOTER*/
+.footer {
+	overflow: hidden;
+	padding-left: 15px;
+	padding-right: 26px;
+	background-color: #e1e1e1;
+	background-image: linear-gradient(to top, #e1e1e1 0%, #e1e1e1 100%);
+}
+.footer_item{
+	float: left;
+	display: block;
+	padding: 16px 10px 12px;
+	margin-right: 70px;
+	color: #2c2c2c;	
+}
+.footer_item:hover {
+	text-decoration: underline;	
+}
+.copyright {
+	float: right;
+	margin-top: 16px;
+	margin-bottom: 12px;
+	color: #2c2c2c;	
+}
+/*additional footer*/
+.inline-footer{
+	margin-top: 40px;
+	padding-left: 15px;
+	padding-right: 26px;
+	background-color: #e1e1e1;
+	background-image: linear-gradient(to top, #e1e1e1 0%, #e1e1e1 100%);
+}
+.inline-footer_list {
+	font-size: 0px;
+	display: inline-block;
+}
+.inline-footer_item{
+	display: block;
+	font-size: 13px;
+	padding: 16px 10px 12px;
+	margin-right: 70px;
+	color: #2c2c2c;		
+}
+.inline-footer_item:hover {
+	text-decoration: underline;	
+}
+.inline-footer::after {
```

Comment:
it is better to have one class .clearfix for all of them. I've seen this styles above for another menu